pkg/rancher/wait.go

```diff
@@ -95,3 +95,31 @@ func ToUpdateClientSecretInstruction(imageOverride, systemDefaultRegistry, k8sVe
 		Command:    cmd,
 	}, nil
 }
+
```

Comment:
we could just make it one instruction 

```
(⎈|local:default)➜  ~ k rollout restart deploy/fleet-controller -n cattle-fleet-system
deployment.apps/fleet-controller restarted
```

Reply:
Thanks @ibrokethecloud , I use two instructions to make sure when we manipulate the secret, the fleet-controller doesn't run (in case it starts to deploy..).